stdlib/ballerina-websub/src/main/ballerina/websub/commons.bal

```diff
@@ -407,31 +407,16 @@ public function WebSubHub::publishUpdate(string topic, json payload) returns err
         error webSubError = {message:"Internal Ballerina Hub not initialized or incorrectly referenced"};
         return webSubError;
     } else {
```

Comment:
Doesn't require else block explicitly. Isn't it?

Reply:
Fixed this and other similar occurrences in the file with fd6e70f